harness/setup.py

```diff
@@ -31,6 +31,11 @@
         "tf-114-cpu": ["tensorflow==1.14.0"],
         "pytorch-14-cuda100": ["torch==1.4.0+cu100", "torchvision==0.5.0+cu100"],
         "pytorch-14-cpu": ["torch==1.4.0", "torchvision==0.5.0"],
+        "tf-240-cuda110": ["tensorflow-gpu==2.4.0"],
```

Comment:
Are supporting both TF 1 and TF 2? Why do we have two different versions here?

Reply:
Ah yeah - I meant to come back to this because I actually wasn't sure exactly how this was being used and if it made sense to leave the old versions here. We were and still do ship images with both TF1 and TF2 (TF1 is the default). I see we previously didn't have even the old TF 2 release listed here, so this change may be unnecessary.

Reply:
I got some context on what extras_require is for, and it's as a convenience for people who will `pip install` this who also want to get our recommended versions of the frameworks. So I've removed the old ones, and added new entries that map to each of the latest images, including CUDA 11. Just pushed a new commit...